### ISSUE TITLE
add Email field to commit process

### ISSUE THREAD

**Issue by michaeljokf** [okfde/offenerhaushalt.de#92]
We need to have some kind of a contact information for the person committing data, because some of those commits have errors and in order to fix them we might have to contact the user. 
Maybe we can add an "email" field before the data upload, where the user manually inputs its data? 

German text below "Email" could be: "Fügen Sie Ihre Email hinzu, damit wir Sie bei Problemen kontaktieren können."

**Comment by monsieurhannes** [comment]
In general it was quite strange for me to send the data and then the popup came but nothing else. So I accidentally sent it twice - sorry for that!
Maybe it would work by chaning the color of the button?

**Comment by michaeljokf** [comment]
yeah we will need to make some additional changes here! Changing the color of the button could work, maybe some more or faster responsiveness as well. We will work on it!
